lib/stack_master/sparkle_formation/compile_time/parameters_validator.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'value_validator_factory'
+
+module StackMaster
+  module SparkleFormation
+    module CompileTime
```

Comment:
You can chain these together, like so:

```
module StackMaster::SparkleFormation::CompileTime
...
end
```

Reply:
Other classes are like this. Just following convention :)

Reply:
The nesting though makes this pretty aweful looking...

Reply:
Sticking with nesting modules. Consistent with other classes. Purtiness is in the eye of the beholder 😄